src/components/details/_details.scss

```diff
@@ -43,10 +43,10 @@
   }
 
   .govuk-c-details__text {
-    // TODO: Make margins underneath components consistent
-    margin-bottom: em($govuk-spacing-scale-3, 19px);
-    padding: em($govuk-spacing-scale-3, 19px);
+    margin-left: 3px;
```

Comment:
Has this change been tested across all browsers? I'm wary of trying to align with the toggle as I think it's part of the default browser styling for the details component, and will vary between browsers?

If it does work, it would be good to add a comment to explain what this is doing.

Reply:
Well it appears we've broken it in Firefox anyway...
<img width="745" alt="screen shot 2017-12-06 at 17 01 47" src="https://user-images.githubusercontent.com/23356842/33674380-38485ea6-daa7-11e7-8bf5-f6ff025a1f4c.png">